adapters/consors-bank-adapter/src/test/java/de/adorsys/xs2a/adapter/consors/ConsorsAccountInformationServiceTest.java

```diff
@@ -2,25 +2,36 @@
 
 import de.adorsys.xs2a.adapter.api.*;
 import de.adorsys.xs2a.adapter.api.http.HttpClient;
-import de.adorsys.xs2a.adapter.api.model.Aspsp;
-import de.adorsys.xs2a.adapter.api.model.OK200TransactionDetails;
-import de.adorsys.xs2a.adapter.api.model.RemittanceInformationStructured;
-import de.adorsys.xs2a.adapter.api.model.TransactionsResponse200Json;
+import de.adorsys.xs2a.adapter.api.http.Request;
+import de.adorsys.xs2a.adapter.api.model.*;
 import de.adorsys.xs2a.adapter.impl.http.AbstractHttpClient;
 import de.adorsys.xs2a.adapter.impl.link.identity.IdentityLinksRewriter;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
 
 import java.io.ByteArrayInputStream;
+import java.util.HashMap;
+import java.util.Map;
 
+import static de.adorsys.xs2a.adapter.api.RequestHeaders.PSU_ID;
 import static java.util.Collections.singletonMap;
 import static org.assertj.core.api.Assertions.assertThat;
 
 class ConsorsAccountInformationServiceTest {
 
+    private static final String EMPTY_STRING = "";
```

Comment:
Why do you need a constant for the empty string? It's longer (less readable) than the actual literal and by definition cannot be changed to some other value in the future.

Reply:
removed